tests/e2e/allowlists/allowlists_service_invocation_test.go

```diff
@@ -88,25 +89,26 @@ var allowlistsServiceinvocationTests = []struct {
 }
 
 var moreAllowlistsServiceinvocationTests = []struct {
-	in               string
-	remoteApp        string
-	appMethod        string
-	expectedResponse string
+	in                 string
+	remoteApp          string
+	appMethod          string
+	expectedStatusCode int
+	expectedResponse   string
 }{
 	{
 		"Test allow with callee side grpc",
 		"allowlists-callee-grpc",
 		"grpctogrpctest",
+		http.StatusOK,
 		"success",
 	},
-	// TODO: Verified this manually using python-sdk sample and port-forwarding to http port.
-	// 		 This test always returns success. Need to debug further
-	// {
-	// 	"Test deny with callee side grpc",
-	// 	"allowlists-callee-grpc",
-	// 	"httptogrpctest",
-	// 	"rpc error: code = PermissionDenied desc = access control policy has denied access to appid: allowlists-caller operation: httptogrpctest verb: NONE",
-	// },
+	{
+		"Test deny with callee side grpc",
+		"allowlists-callee-grpc",
+		"httptogrpctest",
+		http.StatusInternalServerError,
```

Comment:
shouldn't this be 403?

Reply:
I think this is test app response code. we need to ensure that we bypass status code. 

When I double-check the conversion, PermissionDenied should be converted to 403.
https://github.com/dapr/dapr/blob/67c4553aea13acef634540469f4a287b759a288e/pkg/messaging/v1/util.go#L243-L244

Reply:
Yes, the test app is doing this